style.css

```diff
@@ -413,14 +422,260 @@ textarea::placeholder {
   opacity: 0.5;
 }
 
+.headerlist {
+  display: none;
+}
+
 @media (min-width: 375px) {
   .content-section {
     background-size: cover;
   }
 }
 
+/* desktop version */
+
 @media (min-width: 768px) {
+  .icon-Union {
+    display: none;
+  }
+
+  .topBar-nav {
+    position: sticky;
+    top: 0;
+    background-color: white;
+  }
+
+  .logo {
+    padding-left: 100px;
+  }
+
+  .headerlist {
+    display: flex;
+    gap: 8px;
+    padding-right: 100px;
+    position: sticky;
+    top: 0;
+    height: 20px;
+    list-style-type: none;
+  }
+
+  .headerlist a {
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+  }
+
+  .headerlist a:hover {
+    color: rgb(24, 155, 160);
+  }
+
   .content-section {
+    padding-top: 50px;
+    background-image: url(img/Header\ bg-desktop.png);
+    background-size: contain;
+    display: flex;
+    height: 718px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    border-bottom-left-radius: 80px 80px;
+  }
+
+  .main-Container {
+    width: 60%;
+    margin-left: 0;
+  }
+
+  .headingTxt {
+    color: #091e42;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    width: 400px;
+  }
+
+  .text-2 {
+    padding: 10px 0;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .work-section {
+    display: grid;
+    grid-template-columns: 1fr;
+    border-radius: 16px;
+    background-color: #f7f7f9;
+    align-items: center;
+    justify-content: center;
+    padding-top: 100px;
+    gap: 40px;
+    margin-bottom: 50px;
+    width: auto;
+    height: auto;
+    padding-bottom: 150px;
+  }
+
+  .work-container {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    background: #fff;
+    border-radius: 24px;
+    border: solid 1px rgb(223, 225, 230);
+    box-sizing: border-box;
+    width: auto;
+    margin-left: 25px;
+    margin-right: 20px;
+  }
+
+  .cards {
+    width: auto;
+    flex-direction: row;
+    gap: 25px;
+    align-items: center;
+    justify-content: center;
+    margin-left: 25px;
+    margin-right: 20px;
+  }
```

Comment:
- [ ] It requested that we stick to the Figma design as much as possible. Therefore let's make sure our projects look almost identical to the figma design.
**HIT: Use this CSS code for the `.cards` class on the desktop version**
```sh
.cards {
    width: 100%;
    flex-direction: row;
    gap: 25px;
}
```